hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -194,6 +194,18 @@ private URL changePrefixFromBlobtoDfs(URL url) throws InvalidUriException {
     return url;
   }
 
+  private URL changePrefixFromDfsToBlob(URL url) throws InvalidUriException {
+    if (url.toString().contains(WASB_DNS_PREFIX)) {
```

Comment:
WASB_DNS_PREIX is ".blob." or just "blob" ??

Reply:
https://github.com/saxenapranav/hadoop/blob/0e76fa384c8091e7ca593b37b4674b41e1c201d1/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemUriSchemes.java#L39
its .blob.